packages/app-builder-lib/src/winPackager.ts

```diff
@@ -269,6 +271,7 @@ export class WinPackager extends PlatformPackager<WindowsConfiguration> {
         throw e
       }
     }
+    return didSign
```

Comment:
This `didSign` flag logic needs to be flipped in order to handle the `continue` within the `catch`
```
let didSign = false
for (let i = 0; i < 3; i++) {
  try {
    await sign(options, this)
    didSign = true
    break
  } catch (e: any) {
    // https://github.com/electron-userland/electron-builder/issues/1414
    const message = e.message
    if (message != null && message.includes("Couldn't resolve host name")) {
      log.warn({ error: message, attempt: i + 1 }, `cannot sign`)
      continue
    }
    throw e
  }
}
return didSign
```